### ISSUE TITLE
Remove redundant else

### ISSUE THREAD

**Issue by gandarez** [code-cracker/code-cracker#355]
When else statement is empty it might raise a diagnostic.

Before:

````csharp
if (true)
{
    //some statement (not a comment)
}
else
{
}
````

After:

````csharp
if (true)
{
    //some statement (not a comment)
}
````

Diagnostic id: `CC0089`
Category: `Usage`
Severity: `Info`

@gandarez is working on it.

**Comment by giggio** [comment]
Ok, you can update the wiki and add the diagnostic id yourself.
This is related to #88, which you are working, right?

I think this is an `Info`, category `Usage`.

@gandarez can you add a test case?

**Comment by gandarez** [comment]
@giggio Yes for sure. I'm saying it's already done. I'm making effort to develop #88

**Comment by giggio** [comment]
Ok, I updated the description. I noticed you updated the wiki, you need to update the issue as well. I did it already. Check the test case, see if you want to add anything.

**Comment by giggio** [comment]
Closed by #358.
